static/css/components/header-bar.less

```diff
@@ -119,6 +119,16 @@
         padding: 10px;
         display: block;
       }
+
+      form {
+        cursor: pointer;
+      }
+
```

Comment:
```suggestion
```
It looks like this is handled by the `cursor` rule that you've added to the button.